src/main/java/com/snowflake/snowpark_java/Functions.java

```diff
@@ -3880,6 +3881,42 @@ public static Column listagg(Column col) {
     return new Column(com.snowflake.snowpark.functions.listagg(col.toScalaColumn()));
   }
 
+  /**
+   * Function to convert column name into column and order in descending manner.
+   *
+   * @since 1.14.0
+   * @param name The input column name
+   * @return Column object ordered in descending manner.
+   */
+  public static Column desc(String name) {
+    return new Column(functions.desc(name));
+  }
+
+  /**
+   * Function to convert column name into column and order in ascending manner.
+   *
+   * @since 1.14.0
+   * @param name The input column name
+   * @return Column object ordered in ascending manner.
+   */
+  public static Column asc(String name) {
+    return new Column(functions.asc(name));
+  }
+
+  /**
+   * Returns the size of the input ARRAY.
```

Comment:
check if this is consistent with: https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/latest/snowpark/api/snowflake.snowpark.functions.array_size

Also make sure the function is exposed both as array_size as well as size

Reply:
yes, both size and array_size are present